modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -21,29 +21,37 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
+Tokenization is the process of tokenizing a value or piece of information, which might be considered sensitive data, into the token that can be mapped back to the original value. 
 
-== Tokenization
+This tokenized information replaces the original value with the token, thus minimizing the risk of matching it to the absolute value, in case a third party accesses the data. For example, this policy may be used to tokenize credit card numbers and map them to completely different ones from the originals.
 
-Tokenization is the process of tokenizing a value or piece of information (presumably sensitive data) into another one (i.e., the token), that can be mapped back onto the original value. +
-This tokenized information, replaces the original value with the token, thus minimizing the risk of matching it to the raw value in case someone happened to read it. For instance, it may be used to tokenize credit card numbers and map them to completely different ones from the originals.
+This policy is designed to work only for Anypoint Runtime Fabric deployments.
```

Comment:
This should be very visible, wouldn't it be best to keep it as a note very visible at the top or as part of the initial table?

Reply:
I agree with Eva, maybe using an IMPORTANT tag can do the trick.

Reply:
Added it to the table at the top of the page. Done! Thanks!